spec/03-plugins/37-opentelemetry/04-exporter_spec.lua

```diff
@@ -140,6 +145,52 @@ for _, strategy in helpers.each_strategy() do
       end)
     end)
 
+    describe("#scoping", function ()
+      lazy_setup(function()
+        bp, _ = assert(helpers.get_db_utils(strategy, {
+          "services",
+          "routes",
+          "plugins",
+        }, { "opentelemetry" }))
+
+        setup_instrumentations("all", {
+          headers = {
+            ["X-Access-Token"] = "token",
+          },
+        })
+      end)
+
+      lazy_teardown(function()
+        helpers.stop_kong()
+        helpers.kill_http_server(HTTP_SERVER_PORT)
+      end)
+
+      it("works", function ()
+        local thread = helpers.http_server(HTTP_SERVER_PORT, { timeout = 10 })
+        local cli = helpers.proxy_client(7000, PROXY_PORT)
+        local r = assert(cli:send {
+          method  = "GET",
+          path    = "/no_plugin",
+        })
+        assert.res_status(200, r)
+        r = assert(cli:send {
+          method  = "GET",
+          path    = "/no_exist",
+        })
+        assert.res_status(404, r)
+
+        -- close client connection
+        cli:close()
+
+        local ok, err = thread:join()
+
+        ngx.sleep(1000)
```

Comment:
use wait_util

Reply:
This is a mistakenly committed debugging code.